ecosystem/web-wallet/src/core/queries/network.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import { LOCAL_FAUCET_URL, LOCAL_NODE_URL } from 'core/constants';
+
+export const getLocalhostIsLive = async () => {
+  try {
+    const localNode = await axios.get(LOCAL_NODE_URL);
```

Comment:
Can these requests be made in parallel?

Reply:
Yeah i can, but its querying localhost which shouldn't have any latency issues...